src/utils.js

```diff
@@ -1,3 +1,19 @@
+const csstree = require('css-tree');
+
+/**
+ * Simple single argument memoize function. Only works if the first and
+ * only argument is a hashable. E.g. a string.
+ */
+function memoize(fn) {
+  const cache = {};
```

Comment:
Probably not an issue in this case, but we can use LRU cache here, like https://github.com/isaacs/node-lru-cache or https://github.com/lukeed/tmp-cache